app/models/course/assessment/question/text_response_comprehension_solution.rb

```diff
@@ -32,4 +34,13 @@ def strip_whitespace_solution
   def strip_whitespace_solution_lemma
     solution_lemma.each(&:strip!)
   end
+
+  def validate_solution_lemma_not_empty
+    errors.add(:solution_lemma, :solution_lemma_empty) if solution_lemma.empty?
```

Comment:
Sorry, I wasn't very clear. The [schema_validations gem](https://github.com/SchemaPlus/schema_validations) will automatically generate this validation because we have `null: false` on `solution_lemma` in the DB schema. You can leave this out.

The problem is surfacing errors on `solution_lemma` when there is a create/update failure, if there are any. To do that, you should be able to tap on the simple_form feature and add something like `f.error :solution_lemma` to the form partial at `_comprehension_solution_fields.html.slim`. You might have to muck with the styling a bit.

Reply:
Have changed it such that I'm no longer defining a new `solution_lemma_empty` error, but replacing the original `blank` error message instead.